test/integration/tasks.test.js

```diff
@@ -166,6 +180,83 @@ describe("Tasks", function () {
           return done();
         });
     });
+
+    it(`Should get all tasks except ${TASK_STATUS.IN_PROGRESS} tasks`, function (done) {
+      chai
+        .request(app)
+        .get(`/tasks/?q=filter%3A${TASK_STATUS.IN_PROGRESS}`)
+        .end((err, res) => {
```

Comment:
In general we use `filter` when we want to filter out the specific things. until I read the description I was in the assumption that we want to filter task with status in progress. can we call the query param `exclude` or something similar. @prakashchoudhary07 @RitikJaiswal75 would like to know your thoughts

Reply:
@heyrandhir is correct here,
I would assume we are filtering here, instead of excluding